tests/TypeInfer.builtins.test.cpp

```diff
@@ -359,7 +359,7 @@ TEST_CASE_FIXTURE(Fixture, "table_insert_correctly_infers_type_of_array_2_args_o
     CHECK_EQ(typeChecker.stringType, requireType("s"));
 }
 
-TEST_CASE_FIXTURE(Fixture, "table_insert_corrrectly_infers_type_of_array_3_args_overload")
+TEST_CASE_FIXTURE(Fixture, "table_insert_correctly_infers_type_of_array_3_args_overload")
```

Comment:
I'm happy to ignore tests if that's a preference, afaict fixing this is harmless (I've actively ignored things like `error'string of words'` -- which annoys most spell checkers -- in tests here, although I believe adding a space wouldn't hurt the tests and would make them more readable).

Reply:
Fixing test case names is fine, FWIW for now we're using the convention of C identifiers for test names - we've used to use boost.test and migrated to doctest earlier this year, so just in case we need to migrate again to a framework that doesn't support string literals as test names we're sticking with identifiers for now.